src/components/search/MovieSection.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+import { showingMovie } from '@utils/showingMovieAtom';
+import { useEffect, useRef, useState, useCallback } from 'react';
+import { useRecoilState } from 'recoil';
+import SearchMovieCard from './SearchMovieCard';
+import { getMovieTopRatedByPageNumber } from '@apis/getMovieNowPlaying';
+
+export default function MovieSection() {
+  const loaderRef = useRef<HTMLDivElement>(null);
+  const searchInputRef = useRef<HTMLInputElement | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [number, setNumber] = useState(2);
+  useEffect(() => {
+    searchInputRef.current = document.querySelector<HTMLInputElement>('#input');
+  }, []);
+
+  const [showingMovies, setShowingMovies] = useRecoilState(showingMovie);
+
+  useEffect(() => {
+    async function fetchMovieData() {
+      const res = await getMovieTopRatedByPageNumber(1);
+      setShowingMovies(res);
+    }
+    fetchMovieData();
+  }, [setShowingMovies]);
```

Comment:
이 부분도 마찬가지로 초기 영화 데이터를 가져오는 부분에 에러 핸들링을 추가하면 예상치 못한 에러에 대응할 수 있을 것 같아요.

```suggestion
useEffect(() => {
  const fetchMovieData = async () => {
    try {
      const res = await getMovieTopRatedByPageNumber(1);
      setShowingMovies(res);
    } catch (error) {
      console.error('Failed to fetch movies', error);
    }
  };
  fetchMovieData();
}, [setShowingMovies]);
```